README.md

```diff
@@ -1,2 +1,243 @@
-# blockstack-ng
-Reference implementation of the Stacks blockchain v2
+# Stacks 2.0
+
+Reference implementation of https://blockstack.org/whitepaper.pdf in Rust.
+
+[![CircleCI](https://circleci.com/gh/blockstack/blockstack-core/tree/develop.svg?style=svg)](https://circleci.com/gh/blockstack/blockstack-core/tree/develop)
+
+## Repository
+
+| Blockstack Topic/Tech | Where to learn more more |
+|---------------------------------|------------------------------------------------------------------------------|
+| Stacks 2.0 | [master branch](https://github.com/blockstack/blockstack-core/tree/master) |
+| Stacks 1.0 | [legacy branch](https://github.com/blockstack/blockstack-core/tree/develop) |
```

Comment:
the legacy branch won't be `develop` -- it should be `stacks-1.0` or something along those lines.

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1251/commits/58690348304e8c9c91e76132e25a06db443883d1